skynet/modules/ttt/summaries/processor.py

```diff
@@ -1,48 +1,58 @@
-import asyncio
-from concurrent.futures import ThreadPoolExecutor
-
 from langchain.chains.summarize import load_summarize_chain
-from langchain.llms import LlamaCpp
-from langchain.prompts import PromptTemplate
+from langchain.prompts import ChatPromptTemplate
 from langchain.text_splitter import RecursiveCharacterTextSplitter
+from langchain_core.documents import Document
+from langchain_openai import ChatOpenAI
 
-from skynet.env import llama_n_batch, llama_n_gpu_layers, llama_path
+from skynet.env import openai_api_base_url
+from skynet.logs import get_logger
 
-from .prompts.action_items import action_items_template
-from .prompts.summary import summary_template
-from .v1.models import Job, JobType
+from .prompts.action_items import action_items_system_message
+from .prompts.summary import summary_system_message
+from .v1.models import JobType
 
-executor = None
 llm = None
+map_reduce_threshold = 12000
+log = get_logger(__name__)
 
 
 def initialize():
-    global executor, llm
-    executor = ThreadPoolExecutor(max_workers=1)
-
-    llm = LlamaCpp(
-        model_path=llama_path,
-        temperature=0.01,
-        max_tokens=4096,
-        n_ctx=4096,
-        n_gpu_layers=llama_n_gpu_layers,
-        n_batch=llama_n_batch,
+    global llm
+
+    llm = ChatOpenAI(
```

Comment:
This is the local app instance, right? Not the "real" OpenAI (or any other compatible service for that matter).

Reply:
yeah..it's just a langchain llm object